skiko/src/awtMain/cpp/windows/OpenGLLibrary.cc

```diff
@@ -0,0 +1,85 @@
+#include <windows.h>
+#include <gl/GL.h>
+#include <jawt_md.h>
+
+namespace OpenGL32Library {
+    typedef void (*PROC_glFinish) (void);
+    typedef void (*PROC_glGetIntegerv) (GLenum pname, GLint *data);
+    typedef const GLubyte *(*PROC_glGetString) (GLenum name);
+    typedef HGLRC (WINAPI * PROC_wglCreateContext) (HDC hDc);
+    typedef BOOL (WINAPI * PROC_wglDeleteContext) (HGLRC oldContext);
+    typedef PROC (WINAPI * PROC_wglGetProcAddress) (LPCSTR lpszProc);
+    typedef BOOL (WINAPI * PROC_wglMakeCurrent) (HDC hDc, HGLRC newContext);
+    typedef HGLRC (WINAPI * PROC_wglGetCurrentContext) (void);
+
+    bool isLoaded = false;
+    HINSTANCE lib = 0;
+
+    PROC_glFinish glFinish;
```

Comment:
Same for these global vars. There are two options here:
1. mark it as `static` (no linker symbol for it, will be available just in this file)
2. Move it inside function (my vote for it). Like
```c
void glFinish(void) {
    typedef void (*PROC_glFinish) (void);
    static PROC_glFinish proc = (PROC_glFinish) GetProcAddress(lib, "glFinish"); // Will be called once and stored between function calls
    proc();
}
```

Reply:
The second looks good, done.